src/attestation_types/verify/mod.rs

```diff
@@ -108,14 +114,23 @@ pub fn verify(quote_bytes: &[u8], trusted_public_pck_chain: &str) -> Result<(),
         .borrow()
         .verify_hash(hashed_reportdata, unhashed_data)?;
 
+    // This verifies that MRENCLAVE from the Report matches the known-good measurement
+    if &report.mrenclave[..] != good_measurement {
+        println!("mrenclave: {:?}", report.mrenclave);
```

Comment:
The caller may not want a crate they use to emit diagnostic information like this. Returning the error on the line below is sufficient, I think.

Reply:
You're totally right. I'm just too scatter-brained to take out my debugs. Fixed. Thanks!